src/components/Editor.vue

```diff
@@ -58,6 +58,7 @@
 				</template>
 				<ContentContainer v-show="contentLoaded"
 					ref="contentWrapper" />
+				<SuggestionsBar v-if="isEmptyContent && contentLoaded" />
```

Comment:
Should only be shown if a markdown file is edited, not for plain text files (e.g. `.txt`)

Reply:
from the call i understood that SuggestionsBar should be shown if markdown file is not edited (empty). What should i use instead?

Reply:
If you create a `.txt` file or upload any other code file (like `.js` or `.php`) and open it we will not have any rich text editing, so the options there do not make sense. There is also no menu bar. Let's just not show any suggestions in this case.